week-3-data-join.html

```diff
@@ -82,202 +79,160 @@
+    {"Department":"Sucre","Region":4,"Victims":335825},
+    {"Department":"Tolima","Region":5,"Victims":190575},
+    {"Department":"Valle Del Cauca","Region":1,"Victims":528749},
+    {"Department":"Vaupes","Region":2,"Victims":6648},
+    {"Department":"Vichada","Region":3,"Victims":7380}
+ 
   ];
-
   var options = {
     sorted: false,
     filtered: false,
   };
-
-  var MAX_ENROLLMENT = 1783;
-
+  var MAX_VICTIMS = 1749491;
   // Select the first table body element within the DOM and save that selection as a new variable, tbody.
   var tbody = d3.select("tbody");
-
-
   function updateTable(){
-
     // Erase any existing html code within tbody by replacing it with a blank string "".
     tbody.html("");
-
     // Binding data - this will be a very common task in our use of D3. Start by making a selectAll of <tr> elements that do not yet exist (they will soon!).
     // Then we bind our data (schools) with .data and .enter()
     // Lastly, we append the <tr> elements, which are become our current selection because of the prior selectAll("tr")
     var rows = tbody.selectAll("tr")
-      .data(schools)
+      .data(conflict)
       .enter()
       .append("tr");
-
     // If you run, console.log(rows), you should now see an array of <tr> elements printed out to the console, each of which should contain a __data__ property containing one object from our JSON array.
     // console.log(rows);
-
     // Take the rows variable and append a <td> elements containing d.name - the name of each schools for each bound data element.
     rows.append("td")
-      .html(function(d) { return d.name });
-
+      .html(function(d) { return d.Department });
     // Same as above, except for level of the school.
     rows.append("td")
-      .html(function(d) { return d.level_text });
-
+      .html(function(d) { return d.Region });
     // Same as above, except for the number of students enrolled in the school.
     rows.append("td")
-      .html(function(d) { return d.enrollment + ' students' });
-
+      .html(function(d) { return d.Victims + ' victims' });
     // Same as above, calculating a percent of the space allowed that the div should fill:
     rows.append("td")
       .append("div")
       .classed("bar", true)
-      .style("width", function(d) { return ((d.enrollment / MAX_ENROLLMENT) * 100) + '%' })
-
+      .style("width", function(d) { return ((d.Victims / MAX_VICTIMS) * 100) + '%' })
   };
-
   updateTable();
-
-
   // We can use d3.on to trigger events - note that these just change the class of the buttons when clicked, they do not trigger any other events. Still, you should be able to click on the buttons and see them change (become darker as through depressed).
   
-  d3.select("#sort-asc")
-    .on("click", function(){
-      d3.select('#sort-desc').classed('active', false);
-      d3.select('#sort-asc').classed('active', true);
-    });
-
-  d3.select("#sort-desc")
-    .on("click", function(){
+  //SORT DATA 
```

Comment:
Why here? Note that this is outside of the updateTable() function, so it will only run once, when we first load the script. But of course, we want to sort/filter every time updateTable() is called.

Finally, hope this came through in the class discussion at the start of class, but we can continue to use the transform code that we created last week. Refer to the email I sent out earlier in the week for more details.